invenio_vocabularies/contrib/subjects/jsonschemas/subjects/subject-v1.0.0.json

```
@@ -30,6 +30,22 @@
       "description": "Human readable label in different languages.",
       "$ref": "local://vocabularies/definitions-v1.0.0.json#/title"
     },
+    "props": {
+      "type": "object",
+      "patternProperties": {
+        "^.*$": {
+          "type": "string"
+        }
+      }
+    },
+    "identifiers": {
+      "description": "Alternate identifiers for the subject.",
+      "type": "array",
+      "items": {
+        "$ref": "local://definitions-v1.0.0.json#/identifiers_with_scheme"
```

```suggestion
        "$ref": "local://definitions-v2.0.0.json#/identifiers_with_scheme"
```

There's a [new version apparently](https://github.com/inveniosoftware/invenio-records-resources/blob/master/invenio_records_resources/records/jsonschemas/definitions-v2.0.0.json), but we're pretty bad at enforcing it and namespacing it correctly...